### PR TITLE
[11.0][ADD] crm_stage_type

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
CRM Stage Type
==============

This module extends the functionality of the CRM adding a type in the stages.

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
@jbeficent

File: crm_stage_type/models/crm_stage.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class Stage(models.Model):
+    _inherit = 'crm.stage'
+
+    lead_type = fields.Selection(
```
Comment (JordiBForgeFlow) [inline crm_stage_type/models/crm_stage.py:10]: add old_name='type'

Comment (MiquelRForgeFlow) [reply crm_stage_type/models/crm_stage.py:10]: done

**Review by LoisRForgeFlow — APPROVED** [review]
Functional review, LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]
LGTM :+1:

**Pull request closed without merging**